lib/src/main/java/io/ably/lib/http/Http.java

```diff
@@ -424,6 +424,9 @@ public void finalize() {
 			}
 			if(!acceptSet) { conn.setRequestProperty(ACCEPT, JSON); }
 
+			/* pass required headers */
+			conn.setRequestProperty(HttpUtils.X_ABLY_LIB_HEADER, HttpUtils.getAblyLibValue());
```

Comment:
We should statically format the header string once and use that value each time the header is added, instead of formatting the string each time there is a request.